lib/tooling/llvm-mca-tool.ts

```diff
@@ -47,21 +47,64 @@ export class LLVMMcaTool extends BaseTool {
 
     override async runTool(compilationInfo: Record<any, any>, inputFilepath?: string, args?: string[]) {
         const isets = new InstructionSets();
-        const target = isets.getInstructionSetTarget(compilationInfo.compiler.instructionSet);
-        if (target != null) args?.push('-mtriple=' + target);
+        let target = isets.getInstructionSetTarget(compilationInfo.compiler.instructionSet);
+        const prependArgs: string[] = [];
 
+        // Check if compiler target is overridden with --target.
```

Comment:
Clang accepts arguments in two forms, `--target=x86_64-pc-linux-gnu` and `-target x86_64-pc-linux-gnu`

This quickly becomes more and more complicated 😅

Reply:
Addressed.